src/Application.php

```diff
@@ -32,120 +35,117 @@
 class Application
 {
     /**
-     * @var string
+     * @var LoopInterface
      */
-    private $rootPath;
+    private $loop;
 
     /**
-     * @var string
+     * @var RequestHandler
      */
-    private $host;
+    private $requestHandler;
 
     /**
-     * @var int
+     * @var ServerContext
      */
-    private $port;
+    private $serverContext;
 
     /**
      * @var string
      */
-    private $environment;
+    private $rootPath;
 
     /**
-     * @var bool
+     * @var string
      */
-    private $debug;
+    private $bootstrapPath;
 
     /**
-     * @var bool
+     * @var Kernel
+     *
+     * Kernel
      */
-    private $silent;
+    private $kernel;
 
     /**
-     * @var string
+     * @var HttpServer
```

Comment:
If we don't stop the application, there is no need to store the socket and HTTP server as instance variables. What do you think? Maybe we can return to local variables?